src/api.py

```
@@ -0,0 +1,390 @@
+        return APIResponse(
+            success=True,
+            message="Data exported successfully",
+            data={
+                'format': format,
+                'export_time': datetime.utcnow().isoformat(),
+                'data': exported_data
+            }
+        )
+    except Exception as e:
+        logger.error(f"Error exporting data: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)